test/jasmine/tests/calcdata_test.js

```diff
@@ -985,6 +986,7 @@ describe('calculated data and points', function() {
                                 expect(gd._fullLayout[trace.type === 'splom' ? 'xaxis' : axName]._categories).toEqual(finalOrder, 'wrong order');
                             }
                         })
+                        .then(delay(100))
```

Comment:
cc @antoinerg i haven't seen `test-jasmine` fail since I added this in 10+ runs :tada: